### ISSUE TITLE
Release on addons.mozilla.org

### ISSUE THREAD

**Issue by carbongreat13** [mullvad/browser-extension#115]

**Comment by ruihildt** [comment]
This was planned, but no tracked.

We need to release it on https://addons.mozilla.org in order to enable auto-update for users.

**Comment by ruihildt** [comment]
@carbongreat13 We plan to host the auto-updates on our servers, would that solve your issue? Or is there a more specific reason why you would like to have the extension on AMO (addons.mozilla.org)?

**Comment by Victor239** [comment]
Another reason to have it on AMO is so that regular Firefox desktop users can install it. Mullvad Browser is my secondary browser rather than primary due to limitations like [permanent Private Browsing mode](https://github.com/mullvad/mullvad-browser/issues/29) so it'd be good to have the extension available in both browsers.

**Comment by ruihildt** [comment]
> Another reason to have it on AMO is so that regular Firefox desktop users can install it. Mullvad Browser is my secondary browser rather than primary due to limitations like [permanent Private Browsing mode](https://github.com/mullvad/mullvad-browser/issues/29) so it'd be good to have the extension available in both browsers.

@Victor239 You can install it in any modern Firefox based browser from this page:
https://mullvad.net/en/download/browser/extension

**Comment by Victor239** [comment]
Ah I see, I didn't see a link in the README so I thought the only way was the temporary install.

**Comment by ruihildt** [comment]
> Ah I see, I didn't see a link in the README so I thought the only way was the temporary install.

Good point, I'll make it clearer. Temporary install is only if you develop the extension.

**Comment by begin-theadventure** [comment]
I think the add-ons store could be more trustworthy and accessible for some people, it also has reviews.

But anyway, what's the reason that you want to host it on your site instead of just releasing it on the official add-ons site?

**Comment by ruihildt** [comment]
> But anyway, what's the reason that you want to host it on your site instead of just releasing it on the official add-ons site?

In terms of security, it has no impact since it anyway needs to be signed by Mozilla.

The only reason to keep it outside of AMO would be if we wanted to diverge the code between what's shipped in Mullvad Browser and what's available in AMO.

We don't have such plan at the moment, so I think we may keep both for some time, and then see where we're at in the future.

**Comment by ruihildt** [comment]
@git70 We plan to add it to AMO, it has just not been prioritized.

I can also answer for the Mullvad Browser: we are looking at packaging options and what we're able to maintain.

**Comment by ruihildt** [comment]
I can't speak for the app team, but I see you've already been active in https://github.com/mullvad/mullvadvpn-app/issues/3049. ;)
